ragaai_catalyst/tracers/agentic_tracing/upload/trace_uploader.py

```
@@ -328,13 +451,104 @@ def get_task_status(task_id):
+        }
+    # with _futures_lock:
+    pending_count = len([f for f in _futures.values() if not f.done()])
+    completed_count = len([f for f in _futures.values() if f.done() and not f.exception()])
+    failed_count = len([f for f in _futures.values() if f.done() and f.exception()])
+
+    # Try to get thread pool queue size (may not be available in all Python versions)
+    queue_size = 0
+    try:
+        if hasattr(executor, '_work_queue'):
+            queue_size = executor._work_queue.qsize()
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)